docs/conf.py

```diff
@@ -76,13 +78,15 @@
     "sphinx.ext.ifconfig",
     "sphinx.ext.mathjax",
     "sphinx.ext.napoleon",
+    "sphinx.ext.extlinks",
+    "recommonmark",
 ]
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
 
```

Comment:
Just curiosity... Is AutoStructify setup required here? To be sincere I don't know exactly what it does...

Reply:
Good point. I added it as it allows as it activates some more useful features like inline math in md docs.

Reply:
That is great @FlorianWilhelm, feel free to merge the commit when you are satisfied :)